pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +79,32 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+type Percentage float64
+type ResourceThresholds map[v1.ResourceName]Percentage
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type LowNodeUtilizationArgs struct {
+	metav1.TypeMeta
+
+	UseDeviationThresholds     bool
+	Thresholds                 ResourceThresholds
```

Comment:
It's acceptable to still use `api.ResourceThresholds` for both `Thresholds` and `TargetThresholds` to reduce introduction of new types. Once all the strategies are migrated to the plugin, the next step is to implement conversion from `v1alpha1` configuration to `v1alpha2` configuration. At which point we will revisit the types in both versions. For now, there's no `v1alpha2` configuration yet.

The same holds for `HighNodeUtilizationArgs`